harness/determined/experimental/_native.py

```diff
@@ -392,3 +382,49 @@ def create_trial_instance(
     env, rendezvous_info, hvd_config = det._make_local_execution_env(False, config, hparams)
     trial_context = trial_def.trial_context_class(env, hvd_config)
     return trial_def(trial_context)
+
+
+def submit(
+    context_dir: str,
+    config: Optional[Dict[str, Any]] = None,
+    test: bool = False,
+    master_url: Optional[str] = None,
+    follow_logs: bool = False,
+) -> Any:
+    determined_common.set_logger(
+        util.debug_mode() or det.ExperimentConfig(config or {}).debug_enabled()
+    )
+
+    config, exp_context, master_url = _construct_experiment(context_dir, config, master_url)
+    logging.info(f"Creating an experiment with config: {config}")
+
+    if test:
+        return api.create_test_experiment_and_follow_logs(master_url, config, exp_context)
```

Comment:
This will always follow logs. I guess you should refactor `api.experiment. create_test_experiment_and_follow_logs` and `create_experiment_and_follow_logs` to accept a `follow_logs` bool

Reply:
We should refactor that out later once we decide what should come out of experimental